tests/test_jwt_utils.py

```diff
@@ -78,15 +74,22 @@ def test_missing_correct_audiences(self, caplog: pytest.LogCaptureFixture) -> No
             decode_jwt(jwt_token)
         assert "Failed decoding JWT with any of JWK - details" in caplog.text
 
-    @patch("lbz.jwt_utils.PUBLIC_KEYS", [])
-    def test_empty_public_keys(self) -> None:
-        with pytest.raises(RuntimeError):
-            decode_jwt("x")
-
     def test_validate_missing_iss_exception(self) -> None:
         with pytest.raises(SecurityError, match="'exp'"):
             validate_jwt_properties({"allow": "*", "deny": {}})
 
+    @patch.dict(environ, {}, clear=True)
+    def test_empty_public_keys(self) -> None:
+        with pytest.raises(MissingConfigValue, match="ALLOWED_PUBLIC_KEYS"):
```

Comment:
This is the message you expect to see:
```suggestion
        with pytest.raises(MissingConfigValue, match="'ALLOWED_PUBLIC_KEYS' was not defined."):
```